assets/main.js.liquid

```diff
@@ -80,17 +86,124 @@ Site.Product = {
       infinite: true,
       speed: _this.sliderSpeed,
       slidesToShow: 1,
+      centerMode: true,
+      variableWidth: true,
+      focusOnSelect: true,
+      prevArrow: '.gallery-prev',
+      nextArrow: '.gallery-next',
     });
   },
 
   optionSelect: function() {
     var _this = this;
 
-    new Shopify.OptionSelectors("product-select", { product: productJson, onVariantSelected: _this.selectCallback });
+    _this.selectsStyled = false;
+
+    new Shopify.OptionSelectors("product-select", { product: productJson, onVariantSelected: _this.selectCallback.bind(this) });
```

Comment:
When does this callback trigger?

Reply:
once on init (which triggers the styling function), then every time a change is made to the value of the select